TeknoParrotUi.Common/TeknoParrotUi.Common.csproj

```diff
@@ -196,7 +196,668 @@
     <Content Include="Descriptions\*.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="GameProfiles\*.xml">
```

Comment:
Why change this?

Reply:
I haven't actually made any physical changes to that file. All I can gjinknof is that it's changing because I'm adding the xml in the wrong way. The way I done was to right click the folder in the solution view and add existing item. The choose the file. I assume it's changing the file to accommodate the new file but not 100% sure why it's deleting lines. I might be adding the xml wrongly but that's the only way I can see how to do it. When I made my changes the first time I added the xml and when I looked today there was 2 xml with the same name. So I removed both and added the new revised one this morning. Will try again after work later.

Reply:
Yeah you're adding xml in the wrong way then, just add the new xml into the gameprofiles folder (not thru visual studio)